client/static/css/main.css

```diff
@@ -1,25 +1,25 @@
 /* styles introduced by namex-ui-enhancements code-with-us */
 
 @font-face {
-  font-family: 'OpenSans';
+  font-family: 'BCSans';
   font-weight: 400;
-  src: url('../fonts/open-sans/OpenSans-Regular.ttf');
+  src: url('../fonts/bc-sans/BCSans-Regular.ttf');
```

Comment:
That's odd. Four WOFF files are added in this PR but here the reference is to TTF files.

I recommend these be changed to WOFF (and WOFF2), which are optimized for modern browsers. Example:
https://github.com/bcgov/business-create-ui/blob/da5f03a799bc4441576ee6d46e9642d2c222f71c/src/assets/styles/base.scss#L7

Reply:
done.

Reply:
I checked out your code and ran it locally -- looks good.